project/BuildSettings.scala

```diff
@@ -48,11 +50,14 @@ object BuildSettings {
   }
 
   val fileHeaderSettings = Seq(
-    headerEmptyLine := false,
     excludeFilter in (Compile, headerSources) := HiddenFileFilter ||
-      fileUriRegexFilter(".*/netty/utils/.*") || fileUriRegexFilter(".*/inject/SourceProvider.java$") ||
+      fileUriRegexFilter(".*/cookie/encoding/.*") || fileUriRegexFilter(".*/inject/SourceProvider.java$") ||
```

Comment:
Needs to be reverted, the path changed in the `2.7.x` branch, but not in `2.6.x`:
```suggestion
      fileUriRegexFilter(".*/netty/utils/.*") || fileUriRegexFilter(".*/inject/SourceProvider.java$") ||
```

Reply:
Oh, damn, I can't accept the suggestion because the PR is originally authored by @renatocaval (and pushed to a branch on his repo).

Got an error from GitHub: `You do not have permission to push to this repository` 🤷🏼‍♂️

Reply:
Meh, did the change locally and preserved attribution.